distribution/docs/src/main/jdocs/content/_security-framework/security-encryption-intro-contents.adoc

```diff
@@ -0,0 +1,12 @@
+:title: Security Encryption
+:type: securityFramework
+:status: published
+:parent: Security Framework
+:children:
+:order: 02
+:summary: Security Encryption.
+
+The ${ddf-security} Encryption application offers an encryption framework and service implementation for other applications to use.
+This service is commonly used to encrypt and decrypt default passwords that are located within the metatype and ${admin-console}.
+
+The encryption service and encryption command, which are based on http://www.jasypt.org/[Jasypt], provide an easy way for developers to add encryption capabilities to ${branding}.
```

Comment:
We don't use jasypt anymore, we use Keyczar

Reply:
updated.